src/test/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://${AQTS_DATABASE_ADDRESS}:${AQTS_DATABASE_PORT}/${AQTS_DATABASE_NAME}?currentSchema=${AQTS_SCHEMA_NAME}
+    username: ${AQTS_SCHEMA_OWNER_USERNAME}
+    password: ${AQTS_SCHEMA_OWNER_PASSWORD}
+    hikari:
+      maximum-pool-size: 2
```

Comment:
The src/main/resources config has a connection check here, not sure if you want it here too?

Reply:
I added it to the "real" connection because I was seeing connection closed logs in aws. Not sure if it is needed there, but shouldn't be needed for the tests.